src/main/resources/assets/forestry/recipes/bee_combs_0_16.json

```diff
@@ -0,0 +1,24 @@
+{
+  "result": {
+    "item": "forestry:bee_combs_0",
+    "data": 16
```

Comment:
Seems like there's a bug in my recipe generation system. A block can only have 15 metas.

Reply:
https://github.com/ForestryMC/ForestryMC/pull/2112